src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,24 @@
+package racingcar.model;
+
+public class Car {
+
+    private final Name name;
+    private final Location location;
+
+    public Car(String carName) {
+        this.name = new Name(carName);
+        this.location = new Location();
+    }
+
+    public void move() {
+        this.location.increase();
+    }
+
+    public Integer getLocation() {
```

Comment:
getLocation 네이밍과 달리 Location을 반환하는 게 아니네요.
네이밍을 바꿔보면 어떨까요?

```suggestion
    public Integer position() {
```

Reply:
지적 감사드립니다. getCurrentPosition() 으로 변경하였습니다.